### PR TITLE
Added recipe for us

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/us/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - states = us.cli.states:main
+
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipes/us/meta.yaml]: Please add the [entry point]( http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points ).

```yaml
  entry_points:
    - states = us.cli.states:main
```

File: recipes/us/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/us/meta.yaml]: Please drop `setuptools`.

File: recipes/us/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/us/meta.yaml]: I think the entry point is there, though. I can strip the newline...

File: recipes/us/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/us/meta.yaml]: Whoops! `setuptools` is needed. From the CI:
```
import: 'us'
Traceback (most recent call last):
  File "C:\Miniconda35-x64\conda-bld\test-tmp_dir\run_test.py", line 26, in <module>
    import us
  File "C:\Miniconda35-x64\envs\_test\lib\site-packages\us\__init__.py", line 1, in <module>
    from . import states
  File "C:\Miniconda35-x64\envs\_test\lib\site-packages\us\states.py", line 143, in <module>
    load_states()
  File "C:\Miniconda35-x64\envs\_test\lib\site-packages\us\states.py", line 67, in load_states
    from pkg_resources import resource_stream
ImportError: cannot import name 'resource_stream'
TESTS FAILED: us-0.9.1-py35_0
```

Comment (jakirkham) [inline recipes/us/meta.yaml]: Good catch! Yep, you're right. Sorry for the mislead.

**Comment by jakirkham** [comment]
Feel free to ping me when CI completes and will merge.

File: recipes/us/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/us/meta.yaml]: Ah ok. Sorry must have missed it.

**Comment by pmlandwehr** [comment]
(ping @jakirkham for merge)

**Pull request merged**